lib/jazzy/sourcekitten.rb

```diff
@@ -240,12 +240,19 @@ def self.deduplicate_declarations(declarations)
       end
     end
 
+    def self.filter_excluded_files(json)
+      excluded_files = Config.instance.excluded
+      json.map do |doc|
```

Comment:
This PR bumps SourceKitten to version 0.4.1, which includes the paths of the files being documented.

Format was previously `[{docs1}, {docs2}, ...]` and now it's `[{file1: {docs1}}, {file2: {docs2}}, ...]`.